doc/monitoring/metrics_reference.rst

```diff
@@ -122,3 +139,76 @@ Requests:
 * ``tnt_net_requests_total`` - number of network requests this instance has handled since instance start
 
 * ``tnt_net_requests_current`` - amount of pending network requests
+
+.. _fibers:
+
+-------------------------------------------------------------------------------
+Fibers
+-------------------------------------------------------------------------------
+
+Provides statistics of fibers. If your app creates a lot of fibers, it can be used for monitoring
+fibers count and memory usage.
+
+* ``tnt_fiber_count`` - number of fibers
+
+* ``tnt_fiber_csw`` - averall fibers context switches number
+
+* ``tnt_fiber_memalloc`` - the amount of memory that is reserved for fibers
+
+* ``tnt_fiber_memused`` - the amount of memory that is used by fibers
+
+.. _operations:
+
+-------------------------------------------------------------------------------
+Operations
+-------------------------------------------------------------------------------
+
+Number of iproto requests this instance has processed, aggregated by request type.
+It can be used to find out which type of operation clients make more often.
+
+Two types of metrics provided for each request type:
+
+* ``tnt_stats_op_total`` - total number of calls since server start
+* ``tnt_stats_op_rps`` - number of calls for the last 5 seconds
```

Comment:
I suggest to not document _rps metrics at all, it's confusing and it's value could be derived from _total